source/braille.py

```diff
@@ -920,9 +927,9 @@ def update(self):
 		self._rawToContentPos = []
 		self._currentContentPos = 0
 		self.selectionStart = self.selectionEnd = None
+		self._skipFormatFieldsWithText = False
 		self._skipFieldsNotAtStartOfNode = False
 		self._endsWithField = False
-

```

Comment:
Hi, there is not any reason for this change. I'm working on this and the 
blank line is reinstated. I thought that this could be removed.
Thanks for your atention on braille and this request.


El 04/07/2016 a las 4:03, James Teh escribió:
> In source/braille.py
> <https://github.com/nvaccess/nvda/pull/5980#discussion_r69402058>:
>
>>  		self._skipFieldsNotAtStartOfNode = False
>>  		self._endsWithField = False
>> -
>
> nit: Is there any reason to remove this blank line? Otherwise, let's
> reinstate it. :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nvaccess/nvda/pull/5980/files/9c23aaa8706e7ddc6e1db05e874e9dde881281b9#r69402058>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APE65HYNE5RPWgR1gb7HjSPTxKMfKYemks5qSGndgaJpZM4IjMQN>.
>